src/components/Slider/Slider.stories.tsx

```diff
@@ -40,3 +43,57 @@ export const Range: ComponentStory<typeof SliderComponent> = (args) => {
 		/>
 	)
 }
+
+const FlexRow = styled.div`
+	display: flex;
+`
+
+export const RangeWithInputs: ComponentStory<typeof SliderComponent> = (
+	args
+) => {
+	const [inputValue, setInputValue] = useState<SliderValue>([60, 80])
+	const [sliderValue, setSliderValue] = useState<SliderValue>([60, 80])
+
+	const startValue = (inputValue as (string | number)[])[0]
+	const endValue = (inputValue as (string | number)[])[1]
+
+	return (
+		<>
+			<FlexRow>
+				<TextInput
+					value={startValue.toString()}
```

Comment:
Btw. handy shortcut for doing this is:
```suggestion
					value={`${startValue}`}
```

Reply:
I understand that it's shorter but TBH, `.toString()` seems to me better in this case because it's explicit what you intended to do.

Reply:
It's a trend I've spotted in the JS/TS world. But your reasoning makes sense 🙂 👍